awx/main/dispatch/worker/base.py

```diff
@@ -35,6 +35,7 @@ class WorkerSignalHandler:
 
     def __init__(self):
         self.kill_now = False
+        signal.signal(signal.SIGTERM, signal.SIG_DFL)
```

Comment:
I like this much better than what I originally suggested about detecting the inherited signal handler from the parent process.